guide/onboarding/introduction.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: guide
+title: Onboarding
+description: An overview of some of the concepts to consider when building onboarding experiences.
+nav_order: 2
+has_children: true
```

Comment:
You can remove `has_children` if there are no sub-pages. With that property enabled, you can expand the section, even when there are no sub pages.